person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age.to_i
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] You should not pass the id in the constructor instead create a random id so they're unique. Please make sure to fix this issue.
  - You can consider using this. `@id = Random.rand(1..1000)`